# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xpp/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur xpp-1.5-orig/xpp.h xpp-1.5/xpp.h
--- xpp-1.5-orig/xpp.h	2004-12-06 17:59:35.000000000 +0000
+++ xpp-1.5/xpp.h	2006-04-24 10:34:35.351933500 +0000
@@ -177,7 +177,7 @@
    */
 
   int /* O - index of the record of the given option in the array */
-  printFiles::getSpecialOptionIndex(int *num_special_opts_in_arr,
+  getSpecialOptionIndex(int *num_special_opts_in_arr,
 				      /* number of options in the array */
 				      special_opt_t **special_opts_arr,
                                       /* array of numerical/string 
